cmd/go.mod

```diff
@@ -44,6 +44,8 @@ require (
 	github.com/checkpoint-restore/go-criu/v5 v5.3.0 // indirect
 	github.com/cilium/ebpf v0.7.0 // indirect
 	github.com/containerd/console v1.0.3 // indirect
+	github.com/containerd/containerd/api v1.8.0-rc.2 // indirect
```

Comment:
This PR shall not be merged with RC version.

Reply:
This PR was raised so as to test the containerd/api module compatiblity with cadvisor.  I am updating as each rc version releases so that we know there are no breaking changes.  

PR needs to be merged only after containerd/api v1.8.0 is released. 

More discussion on the same is happening on this thread https://kubernetes.slack.com/archives/CHGFYJVAN/p1714733633634819

Reply:
Note that API v1.7.19 has been tagged; the v1.8.0-rc is the API version from the main branch (containerd v2.0-dev), so depending on what's needed, maybe the v1.7.19 version is an option?

Reply:
Yes. Thats a better option. Will update to api/v1.7.19

Reply:
Done.

Reply:
Looks like the "indirect" here stuck on 1.8; could be either go modules's tendency to not downgrade once it's there, or could there be something else in the dependency tree pushing it up? 🤔 

(Sorry, maybe you already noticed that, but just in case 🙈 🫶 )

Reply:
My bad. Hadnt checked the new diff before pushing. Thought tidy will downgrade it automatically. It was because of this

> could be either go modules's tendency to not downgrade once it's there

running just a `go mod tidy` will not downgrade, had to remove and run the tidy again so as to bring in 1.7.19 as indirect dependency.

Reply:
Yes, that fooled me many times; if a dependency version (based on the dependency tree "minimum version selection")  either becomes lower, or even is no longer a dependency, then go modules thinks it was a manual override / upgrade.

That sometimes makes sense (if you indeed updated the version because you wanted a newer version) but sometimes you don't, and just remove all indirects to force it re-resolving the minimum versions can help find those.